instance/migrations/0078_remove_openedxinstance_active_appserver.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.3 on 2017-02-15 09:22
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('instance', '0077_populate_is_active'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='openedxinstance',
+            name='active_appserver',
+        ),
```

Comment:
Nit: If you want, you can combine these three migrations into a single migration with three `operations`